### PR TITLE
Add Repetier-Server platform documentation

### PR DESCRIPTION
**Pull request by MTrab**
**Description:**
Documentation for Repetier-Server platform

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19930
git 

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

### CODE REVIEW

**Comment by MTrab** [comment]
## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

File: source/_components/sensor.repetier.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+sensor:
+  - platform: repetier
+    url: http://IP_ADDRESS
+    port: PORT
+    api_key: YOUR_API_KEY
+```
+
+Configuration options for the Repetier-Server Sensor:
+
+- **url** (*Required*): The URL to connect to without the port part.
```
Comment (ludeeus) [inline source/_components/sensor.repetier.markdown]: Please use our new style for defining configuration options :)
<https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration>

**Review by ludeeus — DISMISSED** [review]
Please use our new style for defining configuration options :)
<https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration>

File: source/_components/sensor.repetier.markdown (hunk 1 above)
Comment (MTrab) [reply source/_components/sensor.repetier.markdown]: Done - I think

File: source/_components/sensor.repetier.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+decimals:
+  description: How many decimals to show in state percent. Only used if state_percent is true.
+  required: false
+  type: integer
+  default: 1
+{% endconfiguration %}
+
+### {% linkable_title Retrieve API-key %}
+
+To generate the needed API-key, go into your Repetier Server web-console, push the settings icon (the gear icon) and select User Profiles.
```
Comment (DubhAd) [inline source/_components/sensor.repetier.markdown:47]: This needs to be a list of some form

**Review by DubhAd — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing PR, since parent PR has been closed.

**Pull request closed without merging**